iOSDeviceManager/Devices/PhysicalDevice.m

```diff
@@ -25,14 +25,11 @@ + (PhysicalDevice *)withID:(NSString *)uuid {
     PhysicalDevice* device = [[PhysicalDevice alloc] init];
 
     device.uuid = uuid;
-
-    NSError *err;
-    
-    FBDeviceSet *deviceSet = [[DeviceUtils deviceSet:FBControlCoreGlobalConfiguration.defaultLogger ecidFilter:nil] await:&err];
-    FBDevice *fbDevice = [deviceSet deviceWithUDID:uuid];
+    static FBDevice *fbDevice;
```

Comment:
Why it's static here? You're initializing it without any condition anyway

Reply:
Seems like it remained from one of previous solutions, because this construction was after:
    `dispatch_once(&onceToken, ^{`